### PR TITLE
adds text example of mapper post processor 

### LINKED ISSUE #945: Feature request: editor support for relational database connection post-processor
### Similar requests

- [X] I have searched and found no existing similar requests

### How are you using Studio?

_No response_

### What problems are you trying to solve?

We should add editor support for post-processor part of relational database connection. This should also have an extension mechanism.

### Describe the solution you would like to see implemented

- [ ] `MAYBE` Add a form editor for basic `post-processor`
- [ ] Add extension mechanism for `post-processor`

### Describe alternatives you have considered

_No response_

### Documentation, Design, Adoption, Migration Strategy

_No response_

### Contribution

- [ ] I would like to work on this feature

### PR DESCRIPTION
**Pull request by xannem**
Fixes #945 

Adds text snippet of mapper post processor for users during text mode. 
Kapture 2022-09-13 at 15 31 44

## How did you test this change?

- [ ] Test(s) added
- [X] Manual testing (please provide screenshots/recordings)
- [ ] No testing (please provide an explanation)

### CODE REVIEW

File: packages/legend-application-studio/src/components/editor/edit-panel/GrammarTextEditor.tsx [hunk 1]
```diff
@@ -544,6 +545,11 @@ const getParserElementSnippetSuggestions = (
           description: 'relational database connection',
           insertText: RELATIONAL_DATABASE_CONNECTION_SNIPPET,
         },
+        {
+          text: PURE_CONNECTION_NAME.RELATIONAL_DATABASE_CONNECTION,
+          description: 'post processor with relational database connection',
```
Comment (akphi) [inline packages/legend-application-studio/src/components/editor/edit-panel/GrammarTextEditor.tsx:550]:
```suggestion
          description: 'relational database connection with post-processor',
```

File: .changeset/wise-feet-play.md [hunk 2]
```diff
@@ -1,4 +0,0 @@
----
-'@finos/legend-application-studio': patch
```
Comment (akphi) [inline .changeset/wise-feet-play.md:2]: we don't ever remove changeset, this changeset belongs to another PR right?

**Review by akphi — CHANGES_REQUESTED** [review]

File: .changeset/wise-feet-play.md (hunk 2 above)
Comment (xannem) [reply .changeset/wise-feet-play.md:2]: true 😢

File: .changeset/wise-feet-play.md [hunk 3]
```diff
@@ -1,4 +1,3 @@
 ---
 '@finos/legend-application-studio': patch
 ---
-
```
Comment (akphi) [inline .changeset/wise-feet-play.md:4]: Please revert this change and add changeset for the missing package.

File: packages/legend-application-studio/src/stores/LegendStudioCodeSnippets.ts [hunk 4]
```diff
@@ -166,7 +166,7 @@ export const POST_PROCESSOR_RELATIONAL_DATABASE_CONNECTION_SNIPPET = `Relational
   type: \${3:H2};
   specification: \${4:LocalH2 {\\}};
   auth: \${5:DefaultH2 {\\}};
-  // example of mapper post processor
+  // example of mappeer post processor
```
Comment (akphi) [inline packages/legend-application-studio/src/stores/LegendStudioCodeSnippets.ts:169]:
```suggestion
  // example of mapper post processor
```

File: packages/legend-art/style/base/_panel.scss [hunk 5]
```diff
@@ -226,6 +226,35 @@
...
+    &__actions {
+      width: 20%;
+      display: flex;
+    }
+
+    &-btn {
+      @include flexCenter;
+
+      flex: 2.8rem 0 0;
+      // padding: 0 1rem;
```
Comment (akphi) [inline packages/legend-art/style/base/_panel.scss:248]: if you don't need it, let's remove it. Also, for now we can do `width: 2.8rem` instead

**Review by akphi — CHANGES_REQUESTED** [review]

**Review by akphi — APPROVED** [review]
